tez/tez.sh

```diff
@@ -126,6 +134,23 @@ if [[ "${role}" == 'Master' ]]; then
   # Restart hive server2
   systemctl restart hive-server2
   systemctl status hive-server2  # Ensure it started successfully
-fi
+}
+
+function main() {
+  local role
+  role="$(/usr/share/google/get_metadata_value attributes/dataproc-role)"
+
+  # Let spark continue using the existing hive configuration, as it will
+  # not want to use hive on tez.
+  cp /etc/hive/conf/* /etc/spark/conf/
```

Comment:
Nit: use HIVE_CONF_DIR and create a SPARK_CONF_DIR

Reply:
Ok, here also this should be substituted with proper variables.